src/teleop/gui/src/components/ControlGUI.vue

```diff
@@ -2,7 +2,11 @@
 <div>
     <DriveControls></DriveControls>
     <ArmControls></ArmControls>
-    <GimbalControls></GimbalControls>>
+    <DriveVelDataH></DriveVelDataH>>
+    <DriveVelDataV></DriveVelDataV>>
```

Comment:
Please remove the extra angle brackets at the end of these lines, they just show up as text on the gui